src/Share/BackgroundJobs/Diffs/ContributionDiffs.hs

```diff
@@ -23,52 +23,81 @@ import Share.Utils.Logging qualified as Logging
 import Share.Web.Authorization qualified as AuthZ
 import Share.Web.Errors (EntityMissing (..), ErrorID (..))
 import Share.Web.Share.Diffs.Impl qualified as Diffs
+import System.Clock qualified as Clock
 
--- | Check every 10 minutes if we haven't heard on the notifications channel.
+-- | Check every 30 seconds if we haven't heard on the notifications channel.
```

Comment:
This is fine, just curious why the change; if the PG Notify stuff is working then this is just a fallback :)

Reply:
Oh, I just thought that after a full web server restart it would be odd for a worker to wait for 10 entire minutes before checking deciding to peek at the work queue. But we can revert, it's a corner case either way.

Reply:
Yeah that's true, for that we should just change the startup behaviour, maybe start with a notification on each channel when we first initialize the TVar in `Share.Postgres.Notifications`; that way we check every channel for work immediately on startup :)

Reply:
Agreed, I'll do that and bump this back up to 10 minutes